src/renderers/shared/fiber/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -543,7 +543,7 @@ describe('ReactIncrementalErrorHandling', () => {
     });
   });
 
-  it('continues work on other roots despite caught errors', () => {
+  xit('continues work on other roots despite caught errors', () => {
```

Comment:
Is there any way to still verify this is the case? It used to be very frustrating that breaking one root would fail to schedule others. Perhaps we can rewrite this as a ReactDOM test instead of disabling?

Reply:
👍  I'll address this before landing, mostly pushed this for feedback from @sebmarkbage